test/test_functional_tensor.py

```diff
@@ -323,7 +323,7 @@ def test_adjust_gamma(self):
 
                 rbg_tensor = adjusted_tensor
                 if adjusted_tensor.dtype != torch.uint8:
-                    rbg_tensor = F.convert_image_dtype(adjusted_tensor, torch.uint8)
+                    rbg_tensor = F_t.convert_image_dtype(adjusted_tensor, torch.uint8)
```

Comment:
I wonder if we should revert those changes now that we have the implementation back in `functional.py` ?